src/components/CurrencyLogo/CurrencyLogoComponent.stories.mdx

```diff
@@ -0,0 +1,19 @@
+import { DAI, WETH } from '@swapr/sdk'
+
+import { Canvas, Meta, Story } from '@storybook/addon-docs'
+import { Box, Flex, Text } from 'rebass'
+import { CurrencyLogoComponent } from './CurrencyLogo.component'
+
+import { NATIVE_CURRENCY_LOGO } from './CurrencyLogo.utils'
+
+<Meta title="Currencies" component={Flex} />
+
+# CurrencyLogoComponent
+
+Show the currency logo.
+
+<Canvas>
+  <Story name="CurrencyLogoComponent">
+    <CurrencyLogoComponent currency={WETH[1]} sources={[NATIVE_CURRENCY_LOGO[1]]} size="48px" />
+  </Story>
+</Canvas>
```

Comment:
Good to have this Component! I would also add on to add argTypes so that we can switch between few different logos and also to be able to see what arguments are being passed. Here are docs for the reference there are examples https://storybook.js.org/docs/react/writing-docs/mdx
![Screenshot 2022-11-02 at 13 57 36](https://user-images.githubusercontent.com/33226956/199495625-2c7e8e08-4483-45e4-90a4-6ae320768c27.png)

Reply:
Thanks for the feedback. It's now possible to edit those props.

<img width="492" alt="image" src="https://user-images.githubusercontent.com/5664434/199607468-e1aeea3f-be95-45cf-9282-09553a6a4ad0.png">